trezorctl

```diff
@@ -27,6 +27,7 @@ import functools
 import json
 import os
 import sys
+import trezorlib.stellar as stellar
```

Comment:
please move this to the other trezorlib imports
also: `from trezorlib import stellar`